src/db/migrations/v1.3.0-add-personal-information.js

```diff
@@ -17,6 +17,11 @@ module.exports = {
         "establishment_email",
         Sequelize.STRING
       ),
+      queryInterface.addColumn(
```

Comment:
Don't edit this file.
If we needed a migration to add the column it would have to be in a new script as when deploying to Prod, only new migrations would be run, not old ones.
However, release 12.0.0 removes postgres anyway so no need for this migration.

Reply:
Not sure if we have a new mechanism for migrating cosmos records